test/metabase/api/search_test.clj

```diff
@@ -146,26 +146,26 @@
                         (merge (data-map instance-name)
                                (when-not in-root-collection?
                                  {:collection_id (u/the-id collection)})))]
-    (mt/with-temp [Collection  coll           (data-map "collection %s collection")
-                   Card        action-model   (if in-root-collection?
-                                                action-model-params
-                                                (assoc action-model-params :collection_id (u/the-id coll)))
-                   Action      {action-id :id
-                                :as action}   (merge (data-map "action %s action")
-                                                     {:type :query, :model_id (u/the-id action-model)})
-                   Database    {db-id :id
-                                :as db}       (data-map "database %s database")
-                   Table       table          (merge (data-map "database %s database")
-                                                     {:db_id db-id})
+    (mt/with-temp [Collection coll (data-map "collection %s collection")
+                   Card action-model (if in-root-collection?
+                                       action-model-params
+                                       (assoc action-model-params :collection_id (u/the-id coll)))
+                   Action {action-id :id
```

Comment:
Cursive messed up the with-temp, will have to realign as cljfmt doesn't care ;_;